my-app/src/app/shared/components/log-in-button/log-in-button.component.ts

```diff
@@ -1,8 +1,25 @@
 import { Component } from '@angular/core';
+import { AuthenticationService } from 'src/app/core';
 
 @Component({
   selector: 'app-log-in-button',
   templateUrl: './log-in-button.component.html',
   styleUrls: ['./log-in-button.component.css']
 })
-export class LogInButtonComponent {}
+export class LogInButtonComponent {
+
+  constructor(private auth: AuthenticationService) {}
+
+  onLogin(): void {
+    const userName = this.auth.getUserInfo()?.email;
+    const userPassword = this.auth.getUserInfo()?.password;
+    console.log('User is Logged In');
+    this.auth.login('User_authKey', {email: userName, password: userPassword});
```

Comment:
encapsulate 'User_authKey'' inside the `auth` service. This component doesn't need to know where it gets or stores the info. It just calls the service method)